setup.py

```diff
@@ -73,34 +73,34 @@
     'pytest-cov',
     'pytest-xdist',
     'recommonmark',
-    'rlkit @ git+https://github.com/vitchyr/rlkit/@1d469a509b797ca04a39b8734c1816ca7d108fc8',  # noqa: E501
+    'rlkit @ git+https://github.com/vitchyr/rlkit/@1d469a509b797ca04a39b8734c1816ca7d108fc8',  # pylint: disable=line-too-long; # noqa: E501
     'seaborn',
     'sphinx',
     'sphinx_rtd_theme',
     'yapf==0.28.0',
 ]
 
 with open('README.md') as f:
-    readme = f.read()
+    README = f.read()
 
 # Get the package version dynamically
 with open('VERSION') as v:
-    version = v.read().strip()
+    README = v.read().strip()
```

Comment:
This rename got mixed up
```suggestion
    VERSION = v.read().strip()
```